provider/marathon/marathon_test.go

```diff
@@ -30,6 +30,14 @@ func newFakeClient(applicationsError bool, applications marathon.Applications) *
 	return fakeClient
 }
 
+func assertWithServices(serviceNames []string, expected string, getActual func(string) string) (string, bool) {
+	actual := ""
+	for _, serviceName := range serviceNames {
+		actual = getActual(serviceName)
+	}
```

Comment:
I don't get the for loop: It iterates through all service names and will eventually only store the final one in `actual`. Why is that?